packages/@lwc/integration-karma/test/events/host-element-add-event-listener/index.spec.js

```diff
@@ -0,0 +1,16 @@
+import { createElement } from 'lwc';
+import Test from 'x/test';
+
+it('should support handleEvent', () => {
+    const elm = createElement('x-test', { is: Test });
+
+    let invoked = false;
+    elm.addEventListener('foo', {
+        handleEvent() {
+            invoked = true;
+        },
+    });
```

Comment:
Let's test `removeEventListener` as well, even if we're still doing it on equality of the function rather than the object.